Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/SN3A-Memory-Consumption/memory-consumption

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+set -x
+
+pod() {
+echo $CI_JOB_ID
+###clone e2e-openshift-repo
+echo "cloning e2e-openshift repo*************"
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'git clone -b OpenEBS-base https://github.com/openebs/e2e-openshift.git'
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-openshift && bash Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/SN3A-Memory-Consumption/memory-consumption node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"'
+}
+
+node() {
+job_id=$(echo $1)
+pipeline_id=$(echo $2)
+commit_id=$(echo $3)
+case_id=SN3A
+
+source ~/.profile
+gittoken=$(echo "$github_token")
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+
+bash Openshift-EE/utils/e2e-cr jobname:memory-consumption-jiva jobphase:Waiting init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+bash Openshift-EE/utils/e2e-cr jobname:memory-consumption-jiva jobphase:Running init_time:"$current_time"
+
+echo "*******Deploying Busybox-sts Application****"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-provision-mem-check-jiva metadata="")
+echo $test_name
+cd litmus
+echo "Running the litmus test for Busybox Deployment.."
+
+cp apps/busybox/deployers/run_litmus_test.yml busybox_memcheck.yml
+
+sed -i -e 's/app: busybox-litmus/app: mem-busybox-jiva/g' \
+-e 's/app=busybox-sts/app=busybox-mem-jiva/g' \ #application label
+-e 's/value: statefulset/value: deployment/g' \
+-e 's/value: openebs-cstor-sparse/value: openebs-jiva-default/g' \
+-e 's/value: app-busybox-ns/value: mem-check-jiva/g' busybox_memcheck.yml
+
+cat busybox_memcheck.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:mem-busybox-jiva' job=busybox_memcheck.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:memory-consumption-jiva $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id SN3A 3-functional "Check if the memory consumption doesn't cross the desired threshold for known workload" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+echo "*******Checking Memory Consumption**********"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=memcheck-test-jiva metadata="")
+echo $test_name
+cd litmus
+
+cp apps/prometheus/functional/memcheck/run_litmus_test.yml memcheck_test.yml
```

Comment:
@harshita-sharma-mayadata use a different name for the file to copy the content of run_litmus_test. This is to ensure that cstor and jiva test has to run in parallel.